easybuild/easyconfigs/s/SeqLib/SeqLib-1.2.0-GCC-10.3.0.eb

```
@@ -0,0 +1,49 @@
+# Author: Jasper Grimm (UoY)
+
+easyblock = 'ConfigureMake'
+
+name = 'SeqLib'
+version = '1.2.0'
+
+homepage = 'https://github.com/walaj/SeqLib'
+description = """C++ interface to HTSlib, BWA-MEM and Fermi."""
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+github_account = 'walaj'
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+patches = [
+    '%(name)s-%(version_major_minor)s.0_avoid-bwa-fml-namespace-conflict.patch',
+    '%(name)s-%(version_major_minor)s.0_use-external-deps.patch',
```

templating the name, but hardcoding the patch version, generally makes it easier when creating easyconfigs for newer releases
```suggestion
    '%(name)s-1.2.0_avoid-bwa-fml-namespace-conflict.patch',
    '%(name)s-1.2.0_use-external-deps.patch',
```

---

Done